src/nvim/mouse.c

```diff
@@ -466,6 +491,8 @@ bool do_mouse(oparg_T *oap, int c, int dir, long count, bool fixindent)
   // When dragging or button-up stay in the same window.
   if (!is_click) {
     jump_flags |= MOUSE_FOCUS | MOUSE_DID_MOVE;
+	// Execute the MouseMoved autocmd
+	do_move_autocmd(mouse_row, mouse_col);
```

Comment:
are these the correct places for emitting the event? Check the git log for when `<MouseMove>` was added to see hints for affected code.

need test coverage in `test/functional/ui/mouse_spec.lua`.

Reply:
I believe this is correct for dragging, but I'll definitely look into it more. Could also be removed or used for a separate drag event if that would make more sense.